lib/modules/dosomething/dosomething_auto_translate/dosomething_auto_translate.module

```diff
@@ -0,0 +1,88 @@
+<?php
+
+/**
+ * @file dosomething_auto_translate.module
+ * This module uses the Google Translate API to translate campaign nodes.
+ */
+
+/**
+ * @file dosomething_auto_translate.module
+ *
+ */
+function dosomething_auto_translate_campaigns($start = NULL, $limit = NULL) {
+  $query = new EntityFieldQuery();
+
+  $query->entityCondition('entity_type', 'node')
+    ->entityCondition('bundle', 'campaign')
+    ->propertyCondition('status', NODE_PUBLISHED)
+    ->range($start, $limit);
```

Comment:
If range() gets NULL, it'll remove the range query and execute the query with no limit. I could add a default number to $limit in the case that someone passes in a $start but no $limit.